frame/election-provider-multi-phase/test-staking-e2e/src/lib.rs

```diff
@@ -274,3 +276,60 @@ fn set_validation_intention_after_chilled() {
 			assert_eq!(Nominators::<Runtime>::get(21).unwrap().targets, vec![81]);
 		})
 }
+
+#[test]
+/// Active ledger balance may fall below ED if account chills before unbounding.
+///
+/// Unbonding call fails if the remaining ledger's stash balance falls below the existential
+/// deposit. However, if the stash is chilled before unbonding, the ledger's active balance may
+/// be below ED. In that case, only the stash (or root) can kill the ledger entry by calling
+/// `withdraw_unbonded` after the bonding period has passed.
+///
+/// Related to <https://github.com/paritytech/substrate/issues/14246>.
+fn ledger_consistency_active_balance_below_ed() {
+	use pallet_staking::Error;
+
+	ExtBuilder::default()
+		.staking(StakingExtBuilder::default())
+		.build_and_execute(|| {
+			assert_eq!(Staking::ledger(&11).unwrap().active, 1000);
+
+			// unbonding total of active stake fails because the active ledger balance would fall
+			// below the `MinNominatorBond`.
+			assert_err!(
+				Staking::unbond(RuntimeOrigin::signed(11), 1000),
+				Error::<Runtime>::InsufficientBond
+			);
+			assert_eq!(Staking::ledger(&11).unwrap().active, 1000);
```

Comment:
```suggestion
			assert_noop!(
				Staking::unbond(RuntimeOrigin::signed(11), 1000),
				Error::<Runtime>::InsufficientBond
			);
```

see `assert_noop`, it ensures storage has not change, so the second assert is implied ;)